### ISSUE TITLE
roadmap?

### ISSUE THREAD

**Issue by bhaugen** [open-app/holodex#75]
What is the wild-guess roadmap for holodex?

Do you think of it as a standalone app that will continue to retrieve exported data, as in the current yaml in github repos, or the upcoming spreadsheets?

Or as an app that might visualize org data from (for example) retrieved live from an Enspiral or NRP API in (for example) json?

Or as an app that will also afford creating and maintaining the org data?

Or something else?

(None of this is a hurry. Step by step. Just thinking...)

**Comment by bhaugen** [comment]
The roadmap might also influence how much work you want to put into features that might end up being temporary and then abandoned, vs features that might grow into permanence.

**Comment by ahdinosaur** [comment]
_wild guess_

some parallel streams right now:
- be a sweet way to visualize group data
- be a sweet way for group admins to run instance and pull group data from external sources

then maybe later:
- be a sweet way to integrate group data from other apps
- be a sweet way for group members to sign up and update data using the app

**Comment by simontegg** [comment]
Yeah @ahdinosaur sums it up. These are" ands" rather than "ors" :). 
https://github.com/open-app/holodex/blob/master/comms/value-propositions.md

**Comment by bhaugen** [comment]
Good value props!
